crawl-ref/source/describe.cc

```diff
@@ -4012,22 +4026,36 @@ static string _monster_stat_description(const monster_info& mi)
         vector<string> fast, slow;
         if (!did_speed)
             result << uppercase_first(pronoun) << " ";
-        _add_energy_to_string(speed, me.move, "covers ground", fast, slow);
+        _add_energy_to_string(speed, me.move,
+                              conjugate_verb("cover", plural) + " ground",
+                              fast, slow);
         // since MOVE_ENERGY also sets me.swim
         if (me.swim != me.move)
-            _add_energy_to_string(speed, me.swim, "swims", fast, slow);
-        _add_energy_to_string(speed, me.attack, "attacks", fast, slow);
+        {
+            _add_energy_to_string(speed, me.swim,
+                                  conjugate_verb("swim", plural), fast, slow);
+        }
+        _add_energy_to_string(speed, me.attack,
+                              conjugate_verb("attack", plural), fast, slow);
         if (mons_class_itemuse(mi.type) >= MONUSE_STARTING_EQUIPMENT)
-            _add_energy_to_string(speed, me.missile, "shoots", fast, slow);
+        {
+            _add_energy_to_string(speed, me.missile,
+                                  conjugate_verb("shoot", plural), fast, slow);
+        }
         _add_energy_to_string(
             speed, me.spell,
-            mi.is_actual_spellcaster() ? "casts spells" :
-            mi.is_priest()             ? "uses invocations"
-                                       : "uses natural abilities", fast, slow);
-        _add_energy_to_string(speed, me.special, "uses special abilities",
+            mi.is_actual_spellcaster() ? conjugate_verb("cast", plural) + " spells" :
+            mi.is_priest()             ? conjugate_verb("use", plural) + " invocations"
+                                       : conjugate_verb("use", plural) + " natural abilities", fast, slow);
```

Comment:
These three lines are too long now, right? I wasn't sure how to fix them but keep it readable.

Reply:
I think it'll end up a bit unwieldy no matter what, but the simplest thing is probably just to add a linebreak after each conjugate_verb(), eg:
```
mi.is_actual_spellcaster() ? conjugate_verb("cast", plural)
                             + " spells" :
[...]
```